src/pages/settings/wallet-settings/wallet-settings.html

```diff
@@ -6,7 +6,7 @@
 
 <ion-content>
   <ion-list class="settings-list bp-list">
-    <button ion-item (click)="openWalletName()">
+    <button ion-item (click)="isERCToken ? '' : openWalletName()">
```

Comment:
maybe this is not a good solution since still a button ( with all the buttons effects ) that do nothing. What do you think ?

Reply:
yes I agree, what about this:

```
<div ion-item *ngIf="isERCToken ? ''>
      <div translate float-left>Name</div>
      <ion-note item-end>
        {{wallet && wallet.name}}
      </ion-note>
</div>
```